cmd/cmdtest/testutils.go

```diff
@@ -103,14 +106,58 @@ func TestSetupWithSandbox(t *testing.T, parallel bool) (*TestSandbox, func()) {
 	if err != nil {
 		t.Fatal("Error creating project dir: ", err)
 	}
+
+	t.Log("Created testing project dir: ", sandbox.ProjectDir)
 	t.Log("Created testing config dir: ", sandbox.ConfigDir)
 
 	// Create the config file if it does not already exist.
 	sandbox.ConfigFile = filepath.Join(sandbox.ConfigDir, "config.yaml")
 	data := []byte("home: " + sandbox.ConfigDir + "\n" + "generated-by-tests: Yes" + "\n")
 	err = ioutil.WriteFile(sandbox.ConfigFile, data, 0644)
 	if err != nil {
-		t.Fatal("Error writing config file: ", err)
+		t.Error("Error writing config file: ", err)
+	}
+
+	var outBuffer bytes.Buffer
+	log := &cmd.LoggingConfig{}
+	log.InitLogging(&outBuffer, &outBuffer)
+
+	_, err = cmd.Exists(TestDirPath)
+	if err != nil {
+		t.Errorf("Cannot find source directory %s to copy", TestDirPath)
+	}
+
+	err = cmd.CopyDir(log, TestDirPath, sandbox.TestDataPath)
+	if err != nil {
+		t.Errorf("Could not copy %s to %s - output of copy command %s", TestDirPath, testDir, err)
+	}
+	t.Logf("Directory copy of %s to %s was successful \n", TestDirPath, testDir)
```

Comment:
I think this success message should come after the next line which prints the copy log.